README.md

```diff
@@ -129,6 +129,17 @@ The following is a list of the explainers available in this repository:
 
 * [Introspective Rationale Explainer](http://people.csail.mit.edu/tommi/papers/YCZJ_EMNLP2019.pdf)
 
+## When should I use which?
+|  | Classical Text Explainer | Unified Information Explainer | Introspective Rationale Explainer |
+|---------------|---------|:-------------------:|:----------------------------:|
+| Inherently explainable | Yes | No | No |
```

Comment:
what do you mean by inherently explainable?  if you mean glassbox, then I would think Introspective Rationale Explainer would be yes?  It takes in some sub-network and builds a layer on top of it that is explainable.

Reply:
Yup Ilya is write

model.predict("really really useless long sentence") internals :
> important_words = model.find_important_words("really really useless long sentence)
> important_words == "really useless"
> return actual_model.predict(important_words)

Reply:
I believe the yes no no is correct here. When should I use which? may not have been immediately clear that it is referring to your original model. That might be worth clarifying since it can confuse other users.

If you have an explainable model, it would  not make sense to use introspective rationale explainer since it would add an extra layer in front of your model to make it explainable, but it is already explainable so that will only hurt performance.

Reply:
yes the language is then really confusing.  If this is referring to the model you are bringing, and not what the algorithm is constructing, then this would be correct but the table seems to be more about the outputs rather than the inputs so it would really help to make this clear.

Reply:
How about "Explanation method comparison chart"

Reply:
And edit "Inherently explainable" --> "Input model inherently explainable"

Reply:
For inherently explainable, the best term we could come up with is "lossless explanations", which we used in our readme.  Admittedly, it's a hard concept to convey.